spec/system/consumer/shopping/products_spec.rb

```diff
@@ -147,14 +147,12 @@ def expect_product_description_html_to_be_displayed(product, html, not_include =
                                                       truncate: false, href: false)
     # check inside list of products
     within "#product-#{product.id} .product-description" do
-      expect(html).to include(html)
```

Comment:
I now realise that `html` was supposed to refer to the page html but it got accidentally overridden by a parameter named the same. So I added more commits to show you how I would write it. It's maybe not as *DRY* but much simpler to read. And it makes it easier to fix up all the little mistakes of the previous code.

Reply:
Nice. Thanks @mkllnk. This was a good learning experience :pray:
and I understand what you mean, readability is more important than being pedantically DRY :smile: